cmd/ftl/cmd_deploy.go

```diff
@@ -210,3 +219,25 @@ func findFilesInDir(dir string) ([]string, error) {
 		return nil
 	}))
 }
+
+func setConfigDefaults(config *moduleconfig.ModuleConfig) {
+	switch config.Language {
+	case "kotlin":
+		if config.DeployDir == "" {
+			config.DeployDir = "target"
+		}
+		if len(config.Deploy) == 0 {
+			config.Deploy = []string{"main", "classes", "dependency", "classpath.txt"}
+		}
+		if config.Schema == "" {
+			config.Schema = "schema.pb"
+		}
+	case "go":
+		if config.DeployDir == "" {
+			config.DeployDir = "build"
+		}
+		if len(config.Deploy) == 0 {
+			config.Deploy = []string{"main", "schema.pb"}
+		}
+	}
```

Comment:
@alecthomas I moved the default values out of the scaffold and into here. Also not sure how you want to handle the `schema.pb` here since it's slightly different from the `kotlin` setup.

Reply:
No worries, I'll fix it up when I merge.